lib/resolve/secrets.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const fs = require('fs'),
+    path = require('path'),
+    SECRETS_DIR = '/run/secrets',
+    output = {};
+
+if (fs.existsSync(SECRETS_DIR)) {
```

Comment:
It would be nice to wrap it in a function. This way it would be easier to test.

Reply:
Working on it.